drivers/ieee802154/ieee802154_mcxw.c

```diff
@@ -862,7 +862,7 @@ static int mcxw_cca(const struct device *dev)
 
 	k_sem_take(&mcxw_radio->cca_wait, K_FOREVER);
 
-	return (mcxw_radio->tx_status == OT_ERROR_CHANNEL_ACCESS_FAILURE) ? -EBUSY : 0;
+	return mcxw_radio->tx_status ? -EBUSY : 0;
```

Comment:
just a heads-up this violates a misra rule about branch conditions being bool

Reply:
Fixed. Thank you